RELEASES.md

```diff
@@ -1,3 +1,200 @@
+Version 1.53.0 (2021-06-17)
+============================
+
+Language
+-----------------------
+- [You can now use unicode for identifiers.][83799] This allows multilingual
+  identifiers but still doesn't allow glyphs that not considered characters
+  such as `◆` or `🦀`. More specifically you can now use any identifier that
+  matches the UAX #31 "Unicode Identifier and Pattern Syntax" standard. This
+  is the same standard as languages like Python, however Rust uses NFC
+  normalisation which may be different from other languages.
```

Comment:
Tiny nit: this is the British spelling, American English uses a Z instead. I think I remember someone saying the repo uses American English, but I don't really mind if you want to keep it.
```suggestion
  normalization which may be different from other languages.
```